iree/test/e2e/regression/linalg_quantized_matmul_vs_linalg_matmul.mlir

```diff
@@ -10,12 +10,15 @@
 // Reference: Section 2.3 of https://arxiv.org/abs/1712.05877.
 
 // Equivalent to linalg.quantized_matmul, but not using linalg.quantized_matmul
-func private @quantized_matmul_as_matmul_3x4x5(%lhs : tensor<3x4xi8>, %rhs : tensor<4x5xi8>,  %lhs_zp : i32, %rhs_zp : i32, %acc : tensor<3x5xi32>) -> tensor<3x5xi32> {
+func private @quantized_matmul_as_matmul_3x4x5(%lhs : tensor<3x4xi8>, %rhs : tensor<4x5xi8>,  %lhs_zp : i32, %rhs_zp : i32) -> tensor<3x5xi32> {
```

Comment:
I would like to understand this diff? This is removing an argument from a function - how is that working wrt the current code in this function's body, referencing this value?

Reply:
I think instead of `acc` being passed in from outside, the function itself creates a zero-filled tensor as the argument to the `linalg.matmul`. There is no buffer materialized. The  buffer for the result is reused, i..e it is filled with 0s before the `linalg.matmul`.

Reply:
oh sorry i was just looking at this 1-line diff being commented on here - not the whole picture.
I'm totally ok with the acc zeroing being moved to the callee -- actually that was my first choice, but i ran into (bufferization?) issues with that earlier.

Reply:
IIRC, you were having issues cause you were just using`linalg.init_tensor` and not the `linalg.fill`.

Reply:
ah, right... maybe i just didn't retry after understanding that.